README-APP-ALERTING.md

```diff
@@ -0,0 +1,102 @@
+# Configure Project Alerting
```

Comment:
There are a handful of linting errors to clean up in this file.

https://github.com/google/quota-monitoring-solution/actions/runs/4117222308/jobs/7108259843#step:4:78